TombEngine/Objects/TR5/Trap/tr5_explosion.cpp

```diff
@@ -88,11 +89,12 @@ void ExplosionControl(short itemNumber)
 			if (item->ItemFlags[3])
 			{
 				if (flag == 2)
-					TriggerShockwave(&pos, 48, 32 * item->ItemFlags[2] + 304, 4 * item->ItemFlags[2] + 96, 128, 96, 0, 24, 2048, 0);
-				else
 					TriggerShockwave(&pos, 48, 32 * item->ItemFlags[2] + 304, 4 * item->ItemFlags[2] + 96, 0, 96, 128, 24, 2048, 0);
+				else
+					TriggerShockwave(&pos, 48, 32 * item->ItemFlags[2] + 304, 4 * item->ItemFlags[2] + 96, 128, 96, 0, 24, 2048, 0);
 			}
-			else if (flag == 2)
```

Comment:
Why this condition was reversed? Just asking, not insisting it is an issue.

Reply:
Oh I sometimes revert code to fit the OG code to get a clearer head when I compare them. I reverted it back